webapp/channels/src/components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -213,6 +214,7 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                                     direction={s.direction}
                                     isHeader={s.isHeader}
                                     tabIndex={1}
+                                    intl={this.props.intl}
```

Comment:
0/5 but maybe this should use the injected version of `SubMenuItem` to avoid changes to the snapshots and to remove any chance that the inner and outer items not behave slightly differently because `injectIntl` introduces other subtle differences

Reply:
This was one suggestion I couldn't make work, given the need to reference the injected `SubMenuItem` before it's defined.

Reply:
Huh, that's an interesting problem. Given this is JS, I assume there's some ugly way to make that work, but I'm good leaving this as-is since we want to get rid of the old menu eventually anyway